tests/app/helpers/test_template_helpers.py

```diff
@@ -64,114 +138,87 @@ def test_get_footer_context_nisra_theme(app):
+        )
 
-    assert expected == result
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["page_header"]
 
+    assert result == expected
 
-def test_get_footer_urls_census_cy():
-    base_url = "https://cyfrifiad.gov.uk/"
 
+def test_get_page_header_context_census_nisra(app):
     expected = {
-        "help": f"{base_url}help/sut-i-ateb-y-cwestiynau/help-y-cwestiynau-ar-lein/",
-        "cookies": f"{base_url}cwcis/",
-        "accessibility_statement": f"{base_url}datganiad-hygyrchedd/",
-        "privacy_and_data_protection": f"{base_url}preifatrwydd-a-diogelu-data/",
-        "terms_and_conditions": f"{base_url}telerau-ac-amodau/",
-        "contact_us": f"{base_url}cysylltu-a-ni/",
-        "languages": f"{base_url}help/ieithoedd-a-hygyrchedd/ieithoedd/",
-        "bsl_and_audio_videos": f"{base_url}help/ieithoedd-a-hygyrchedd/hygyrchedd/fideos-hygyrch-gyda-bsl/",
+        "logo": "nisra-logo-en",
+        "logoAlt": "Northern Ireland Statistics and Research Agency logo",
+        "titleLogo": "census-logo-en",
+        "titleLogoAlt": "Census 2021",
+        "customHeaderLogo": "nisra",
+        "mobileLogo": "nisra-logo-en-mobile",
     }
-    result = get_static_content_urls("cy", base_url, "census")
-
-    assert expected == result
-
-
-def test_get_data_layer_nisra_theme():
-    expected = [{"nisra": True}]
-    result = get_data_layer("census-nisra")
-
-    assert expected == result
-
-
-def test_get_census_base_url():
-    expected = "https://census.gov.uk/"
-    result = get_census_base_url(schema_theme="census", language_code="en")
-
-    assert expected == result
-
-
-def test_get_census_base_url_nisra_theme():
-    expected = "https://census.gov.uk/ni/"
-    result = get_census_base_url(schema_theme="census-nisra", language_code="en")
 
-    assert expected == result
-
-
-def test_get_census_base_url_welsh():
-    expected = "https://cyfrifiad.gov.uk/"
-    result = get_census_base_url(schema_theme="census", language_code="cy")
-
-    assert expected == result
-
-
-def test_get_census_base_url_welsh_is_priority_over_nisra():
-    expected = "https://cyfrifiad.gov.uk/"
-    result = get_census_base_url(schema_theme="census-nisra", language_code="cy")
-
-    assert expected == result
-
-
-def test_get_census_base_url_ga():
-    expected = "https://census.gov.uk/"
-    result = get_census_base_url(schema_theme="census", language_code="ga")
+    with app.app_context():
+        survey_config = get_survey_config(
+            theme="census-nisra",
+            language="en",
+        )
 
-    assert expected == result
+        result = ContextHelper(
+            language="en",
+            is_post_submission=False,
+            include_csrf_token=True,
+            survey_config=survey_config,
+        ).context["page_header"]
+    assert result == expected
```

Comment:
Are there tests for context other than the `page_header` or `page_footer`?

Reply:
there aren't, but there is no logic attached to other parts of the context either

Reply:
There are, for example:

```python
self._account_service_url = cookie_session.get(
    "account_service_url", self.context_options.account_service_url
)
```

or 

```python
self._cdn_url = (
    f'{current_app.config["CDN_URL"]}{current_app.config["CDN_ASSETS_PATH"]}'
)
```

Reply:
cdn_url is tested in test_app_create.py, though the other is a valid point, as I don't believe that is tested anywhere else. I'll add a test for this and for the contact_us_url as that also has some logic attached to it

Reply:
Done

Reply:
The testing in test_app_create tests that the `CDN_URL` environment variable is added to the CSP headers i.e. outside of templating. The CDN url that we put in the context here should be also be tested (this also happens to concatenate the cdn url with the assets path).